core/server/helpers/ghost_head.js

```diff
@@ -176,6 +177,11 @@ module.exports = function ghost_head(options) { // eslint-disable-line camelcase
                 escapeExpression(metaData.blog.title) + '" href="' +
                 escapeExpression(metaData.rssUrl) + '" />');
 
+            // don't allow indexing of preview URLs!
+            if (isPreviewPage) {
+                head.push(writeMetaTag('robots', 'noindex,nofollow', 'name'));
```

Comment:
I noticed the `writeMetaTag` utility is available, but isn't used in this specific function - was there any specific reason for that?